_i18n/de.yml

```
@@ -0,0 +1,90 @@
+  release_archive: Release Archiv
+  source_code: Quellcode
+  project: Projekt
+  blog: Neuigkeiten
+  code_of_conduct: Verhaltenskodex
+  governance: Verwaltung
+  teams: Teams
+  priorities: Prioritäten
+  communities: Gemeinschaften
+  resources: Ressourcen
+  asset_library: Asset-Bibliothek
+  press_kit: Presse-Kit
+  showcase: Vorzeigeprojekte
+  education: Bildung
+  foundation: Stiftung
+  about: Über uns
+  donate: Spenden
+  license: Lizenz
+  privacy_policy: Datenschutzerklärung
+  contact_us: Kontakt
+  text: "Juan Linietsky, Ariel Manzur und <a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>Beitragende</a>. Gehostet von der <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot Foundation</a>. Website Quellcode <a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>auf  GitHub</a>."
+sponsors:
+  title: "Godot wird gesponsert durch:"
+  text: Wenn Sie an einem Unternehmensponsoring interessiert sein, schreiben Sie bitte an <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>.<br>Spenden als Einzelpersonen können durch den <a href='https://fund.godotengine.org/'>Godot Engine Entwicklungsfonds</a> getätigt werden.
+  cta: Spenden
+home:
+  h1: Deine kostenlose, Open&#8209;Source<br>Game Engine.
+  subtitle: Entwickle 2D- &amp; 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+  whats_new: Neuigkeiten
+  download_latest: Downloade die neueste Version
+  looking_for: 'Auf der Suche nach <a href="/download/windows/" class="set-os-download-url" data-version="3" title="Download der für Langzeit unterstützten Version von Godot 3">Godot 3</a>, unseren <a href="/download/archive">experimentellen Veröffentlichungen</a>, oder <a href="/download/archive">früheren Versionen</a>?'
+  latest_news: Aktuelle Nachrichten
+  more_news: Mehr Nachrichten
+  cards:
+    h2: Eine andere Art Spiele zu machen
+    design:
+      title: Innovatives Design
+      text: Das Node- und Szenensystem von Godot bietet Ihnen sowohl die Leistung als auch die Flexibilität, alles zu erstellen.
+    script:
+      title: Verwende die richtige Sprache für den Job
+      text: Halte den Code mit einer objektorientierten API unter Verwendung von GDScript, C# und C++ modular, oder bringe  Keep your code modular with an object-oriented API using Godot's own GDScript, C#, C++, or bringe eine eigene durch GDExtension.
+      extra: C# Unterstützung ist derzeit nur für Desktop- und mobile Plattformen ab Godot 4.2 verfügbar.<br>Verwende Godot 3, um C# auf der Webplattform auszuführen.
+    2d:
+      title: Dedizierte 2D Engine
+      text: Make crisp and performant 2D games with Godot's dedicated 2D rendering engine with real 2D pixel coordinates and 2D nodes.
+    3d:
+      title: Einfaches and leistungsstarkes 3D
+      text: Godot's 3D Nodes bieten alles, was zur Erstellung, Animierung und Renderng von 3D-Welten Charakteren benötigt wird.
```

```suggestion
      text: Godots 3D-Nodes bieten alles, was für Erstellung, Animation und Rendering von 3D-Welten und Charakteren benötigt wird.
```
btw we need to check with the german translation/docs team to make sure to use their guidelines for what to translate, haven't kept up where we use the original names

---

I like your approach @mhilbrunner but I feel we then need pronounds then: "die Erstellung, Animation, und das Rendering"